src/Executable.js

```diff
@@ -105,6 +111,63 @@ export default class Executable {
         return this;
     }
 
+    /**
+     * @param {number} minBackoff
+     */
+    setMinBackoff(minBackoff) {
+        if (this._maxBackoff != null && minBackoff >= this._maxBackoff) {
+            throw new Error(
+                "minBackoff cannot be larger than or equal to maxBackoff."
+            );
+        }
+        this._minBackoff = minBackoff;
+    }
+
+    /**
+     * @returns {number | null}
+     */
+    get minBackoff() {
+        return this._minBackoff;
+    }
+
+    /**
+     * @param {number} maxBackoff
+     */
+    setMaxBackoff(maxBackoff) {
+        if (this._minBackoff != null && maxBackoff <= this._minBackoff) {
+            throw new Error(
+                "maxBackoff cannot be smaller than or equal to minBackoff."
+            );
+        }
+        this._maxBackoff = maxBackoff;
+    }
+
+    /**
+     * @returns {number | null}
+     */
+    get maxBackoff() {
+        return this._maxBackoff;
+    }
+
+    /**
+     * @param {number} minBackoff
+     * @param {number} maxBackoff
+     */
+    setBackoff(minBackoff, maxBackoff) {
```

Comment:
Although I do like this method, can we make this private for the time being. I'd like to get this added to Java before we add this method here.